### PR TITLE
32-lint-format-test

### PR DESCRIPTION
**Pull request by rmc3408**
install checkout
node
dependencies
lint
test

### CODE REVIEW

**Comment by rmc3408** [comment]
Passing
image

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest --testTimeout=3000",
     "dev": "ts-node src/server.ts",
-    "lint": "tsc && eslint --ext *.ts,*.json ./src/** && prettier . --write",
+    "lint": "tsc && eslint --ext 'src/*.ts,*.json' && prettier . --write",
```
Comment (sergiobtos) [inline package.json]: I found this on the internet: 

`"lint": "tsc && eslint --ext 'src/*.ts,*.json' --fix && prettier . --write"`

ESLint --fix Flag: It's a good practice to add the --fix flag to the ESLint command. This flag tells ESLint to automatically fix fixable issues in your code where possible. This helps maintain consistent code style and fix minor issues automatically.

Also I found this : 
`src/**/*.ts: Targets TypeScript files recursively within the src/ directory and its subdirectories.`

What do you think about it?

**Review by sergiobtos — APPROVED** [review]
check my frontend review comments and also I added one comment here. other than that looks good to me.

File: package.json (hunk 1 above)
Comment (rmc3408) [reply package.json]: 1. yes. I will add --fix 
2. I was cleaning code... and i removed for testing `**`. Forgot it.

**Pull request merged**